srcpkgs/lilypond/template

```diff
@@ -9,20 +9,18 @@ configure_args="--disable-documentation ac_cv_func_isinf=yes
  --with-texgyre-dir=/usr/share/texmf-dist/fonts/opentype/public/tex-gyre"
 hostmakedepends="autogen automake bison flex fontforge gettext guile1.8 perl
```

Comment:
```
hostmakedepends="autogen automake bison flex fontforge gettext guile1.8 perl
```
guile1.8 is removed with this PR, so it can't be in hostmakedepends

Reply:
Thanks for noticing. :) It’s removed now.